flake.nix

```diff
@@ -94,6 +94,7 @@
           '';
           nativeBuildInputs = [
             pkgs.cargo-watch
+            pkgs.clippy
```

Comment:
We should add a flake check for clippy, but this can happen outside this PR, and after #14 is done.